Dockerfile

```diff
@@ -1,11 +1,14 @@
 # Stage 1: Build frontend assets
-FROM node:20.3.1-alpine as frontend-build
+FROM node:20.2.0-alpine as frontend-build
 
 ENV NODE_ENV production
-COPY .babelrc rollup.config.mjs gulpfile.mjs package.json tailwind.config.mjs yarn.lock ./
-COPY via/static ./via/static
+RUN mkdir -p /tmp/frontend-build
+COPY .babelrc rollup.config.mjs gulpfile.mjs package.json tailwind.config.mjs .yarnrc.yml yarn.lock /tmp/frontend-build
```

Comment:
I just noticed this after after my first review.

Does it make sense that this `COPY` has `/tmp/frontend-build`? Isn't that something we are creating inside docker itself?

Reply:
Ugh, that's the destination. Never mind 😅